pkg/domain/infra/abi/system.go

```diff
@@ -62,6 +58,7 @@ func (ic *ContainerEngine) SetupRootless(_ context.Context, noMoveProcess bool)
 	if err != nil {
 		return err
 	}
+
```

Comment:
nit: extra line (please ignore comment if intentional)

Reply:
fixed and pushed a new version